common/components/Header/components/GasPriceDropdown.scss

```diff
@@ -0,0 +1,17 @@
+@import "../../../../common/sass/variables";
```

Comment:
Sounds good! Webstorm was throwing a warning with this path, but indeed it seems to work fine :+1: